packages/lesswrong/server/search/elastic/ElasticQuery.ts

```diff
@@ -125,6 +129,18 @@ class ElasticQuery {
           },
         });
         break;
+      case "exists":
+        terms.push({
+          bool: {
+            should: [],
+            must: [{
+              exists: {
+                field: filter.field
+              }
+            }]
+          }
+        });
+        break;
```

Comment:
I added this new filter type as well, mostly because if you visit the community map page without selecting a location (like the first time you visit the page) then we don't sort the list by distance and so you might still see users without a map location. This way we don't show those users who didn't opt-in to being on the map.